xml/admin_operating_pools.xml

```diff
@@ -64,825 +64,821 @@
   To organize data into pools, you can list, create, and remove pools. You can
   also view the usage statistics for each pool.
  </para>
- <sect1 xml:id="ceph-pools-associate">
-  <title>Associating pools with an application</title>
 
+ <sect1 xml:id="ceph-pools-operate-add-pool">
+  <title>Creating a pool</title>
   <para>
-   Before using pools, you need to associate them with an application. Pools
-   that will be used with &cephfs;, or pools that are automatically created by
-   &ogw; are automatically associated.
+   A pool can be created as either 'replicated' to recover from lost OSDs by
+   keeping multiple copies of the objects or 'erasure' to get a kind of
+   generalized RAID5/6 capability. Replicated pools require more raw storage,
+   while erasure coded pools require less raw storage. Default is
+   'replicated'.
   </para>
-
   <para>
-   For other cases, you can manually associate a free-form application name
-   with a pool:
+   To create a replicated pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application enable <replaceable>pool_name</replaceable> <replaceable>application_name</replaceable></screen>
-
-  <tip>
-   <title>Default application names</title>
-   <para>
-    &cephfs; uses the application name <literal>cephfs</literal>, &rbd; uses
-    <literal>rbd</literal>, and &ogw; uses <literal>rgw</literal>.
-   </para>
-  </tip>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>pool_name</replaceable> <replaceable>pg_num</replaceable> <replaceable>pgp_num</replaceable> replicated <replaceable>crush_ruleset_name</replaceable> \
+<replaceable>expected_num_objects</replaceable></screen>
   <para>
-   A pool can be associated with multiple applications, and each application
-   can have its own metadata. You can display the application metadata for a
-   given pool using the following command:
+   To create an erasure coded pool, execute:
   </para>
-
-<screen>&prompt.cephuser;ceph osd pool application get <replaceable>pool_name</replaceable></screen>
- </sect1>
- <sect1 xml:id="ceph-pools-operate">
-  <title>Operating pools</title>
-
+<screen>&prompt.cephuser;ceph osd pool create <replaceable>pool_name</replaceable> <replaceable>pg_num</replaceable> <replaceable>pgp_num</replaceable> erasure <replaceable>erasure_code_profile</replaceable> \
+<replaceable>crush_ruleset_name</replaceable> <replaceable>expected_num_objects</replaceable></screen>
```

Comment:
This command should be reviewed, because not all of these parameters are required. In fact, in SES7 I'm pretty sure ony `pool_name` is required: the rest are optional and actually not always needed because SES7 has the Balancer. (There may still be cases when it's desirable to give explicit values for these parameters, but I do strongly believe that this section should emphasize that, thanks to the Balancer, non-expert users don't need to worry about these values anymore.)

Reply:
This is simply what is upstream: https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool 

I will leave this and add a note this requires further review.